types/steam-user/index.d.ts

```diff
@@ -0,0 +1,5456 @@
+// Type definitions for steam-user 4.19
+// Project: https://github.com/DoctorMcKay/node-steam-user
+// Definitions by: vanitasboi <https://github.com/vanitasboi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.0
+// Enums generated from JS by: https://github.com/vanitasboi/JStoTSenum
+
+/// <reference types="node" />
+
+type SteamID = import('steamid');
+type ByteBuffer = import('bytebuffer');
+import EventEmitter = require('events');
+import SteamChatRoomClient = require('./SteamChatRoomClient');
```

Comment:
I believe this requires rewrite, there is no 'SteamChatRoomClient' import (file) in original,
this shuld read:
```ts
import SteamChatRoomClient = require('./components/chatroom');
```
which implies a file at `components/chatroom.d.ts`

Reply:
okay :D